features/storage/kvstore/conf/filesystem/mbed_lib.json

```diff
@@ -0,0 +1,42 @@
+{
+    "name": "storage_filesystem",
+    "config": {
+        "rbp_internal_size": {
+            "help": "If default the size will be 4K*#entries/32",
+            "value": "0"
+        },
+        "rbp_number_of_entries": {
+            "help": "If not defined default is 64",
+            "value": "64"
+        },
+        "internal_base_address": {
+            "help": "If default base address is the first sector after the application code ends.",
+            "value": "0"
+        },
+        "filesystem": {
+            "help": "Options are default, FAT or LITTLE. If default value the filesystem is chosen by the blockdevice type",
+            "value": "default"
+        },
+        "blockdevice": {
+            "help": "Options are default, SPIF, DATAFASH, QSPIF or SD. If default the block device will be chosen by the defined component",
```

Comment:
"help": "Options are default, SPIF, DATAFASH, QSPIF or SD. If default, the block device will be chosen according to the component defined  in targets.json"